docker-compose.yml

```diff
@@ -47,6 +57,32 @@ services:
       - 80:80
     depends_on:
       - scylla-server
+    cpu_shares: 512
+    mem_limit: 1gb
+
+  siren:
+    container_name: siren
+    restart: unless-stopped
+    image: eclipse-mosquitto:latest
+    ports:
+      - 1883:1883
+      - 9001:9001 # why?
+    expose:
+      - 1883
+    volumes:
+      - ./siren-base/mosquitto/mosquitto.conf:/mosquitto/config/mosquitto.conf
+    cpu_shares: 2048
+    mem_limit: 2gb
+    oom_kill_disable: true
+
+    
+
+
+  
+
+
+volumes:
+  db-data:
```

Comment:
why does this have to be so far down at the bottom?

Reply:
The standard specification puts services above volumes as you declare the volumes used by timescale in the service.

Reply:
Here's where I took inspiration.
https://docs.docker.com/compose/compose-file/07-volumes/#example

Reply:
yeah I'm saying why are there like 8 lines between the end of services and this, looks a bit weird

Reply:
Oh lol didn't notice that.  Will shrink it down.